### PR TITLE
Added cms-protractor-helper usage for selecting options

### PR DESCRIPTION
**Pull request by Marketionist**
**This pull request adds cms-protractor-helper usage for selecting options**

Also:
- minor updates for several selectors

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -6,6 +6,7 @@
     },
     "devDependencies": {
         "browserslist-saucelabs": "^0.1.4",
+        "cms-protractor-helper": ">=0.0.3",
```
Comment (vxsx) [inline package.json]: uhm, why not `~0.0.3` ?
eventually we don't want to break things when api stabilizes in cms-protractor-helper and you're ready to release 1.0

Comment (Marketionist) [inline package.json]: Fixed

**Comment by vxsx** [comment]
:+1:

**Comment by mkoistinen** [comment]
LGTM

**Pull request merged**